src/main/java/org/osiam/Osiam.java

```diff
@@ -37,13 +37,15 @@
 import org.springframework.security.authentication.encoding.ShaPasswordEncoder;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.session.jdbc.config.annotation.web.http.EnableJdbcHttpSession;
 
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.Map;
 
 @SpringBootApplication
 @EnableWebSecurity
+@EnableJdbcHttpSession
```

Comment:
This is not needed. Spring Boot has an [Auto Configuration](https://github.com/spring-projects/spring-boot/blob/v1.5.3.RELEASE/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/SessionAutoConfiguration.java) for Spring Session. The Auto Configuration will also make it possible to [define the session timeout](https://github.com/spring-projects/spring-boot/blob/v1.5.3.RELEASE/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/JdbcSessionConfiguration.java#L61) via the [`spring.session` configuration property namespace](https://github.com/spring-projects/spring-boot/blob/v1.5.3.RELEASE/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/SessionProperties.java). But you have to set the configuration property `spring.session.store-type` to `jdbc` so that the Auto Configuration is picked up. See the [reference manual](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-session) for details.